modules/zero-trust-manager-create-route-oidc.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-oidc-federation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="zero-trust-manager-create-route-oidc_{context}"]
+= Creating a route for OIDC
```

Comment:
```suggestion
= Configure external certificate for the managed OIDC discovery provider route
```
And move the route relevant contents in `modules/zero-trust-manager-config-entra-oidc.adoc` to here.